src/components/AppNavigation/EditCalendarModal.vue

```diff
@@ -24,7 +24,19 @@
 					:placeholder="$t('calendar', 'Calendar name …')"
 					@input="calendarNameChanged = true">
 			</div>
-
+			<template v-if="canBeShared">
+				<NcListItem :name="$t('calendar', 'Include calendar in free/busy checks')"
```

Comment:
```suggestion
				<NcListItem :name="$t('calendar', 'All events in this calendar show you as busy by default')"
```

Also document that a transparent calendars VEVENTS will never show you as busy even if the event is set to busy.